Yosemite/Yosemite/Stores/SettingStore.swift

```diff
@@ -6,11 +6,19 @@ import Storage
 // MARK: - SettingStore
 //
 public class SettingStore: Store {
+    private let remote: SiteSettingsRemote
```

Comment:
It might be better to call this `siteSettingsRemote` since there are 2 `Remote` properties in here now.

Reply:
yup! updated in https://github.com/woocommerce/woocommerce-ios/pull/2879/commits/3fd77b03869f2adc5b19f448d2355362c00ccce5